components/loading-box/src/stories.js

```diff
@@ -33,7 +35,8 @@ stories.add('Component default', () => (
   >
     <div style={{ padding: '0 12px' }}>
       <PhaseBanner level="alpha">
-      This part of GOV.UK is being rebuilt &#8211;{' '}
+      This part of GOV.UK is being rebuilt &#8211;
```

Comment:
i'd have expected the indentation of this line to be wrong

Reply:
if you don't like eslint-config-airbnb's styling rules then a good argument to move to prettier :-)